modules/justfiles/README.md

```diff
@@ -0,0 +1,39 @@
+# `justfiles`
+
+The `justfiles` module allows for easy `.just` files importing. It can be useful for example to separate DE specific justfiles when building multiple images.
+
+:::note The module is compatible only with Universal Blue images. :::
+
+## What it does
+
+1. The module checks if the `config/justfiles/` folder is present.
+    
+    * If it's not there, it fails.
+
+2. The module finds all `.just` files inside of the `config/justfiles/` folder or starting from the relative path specified under `include`.
+    
+    * If no `.just` files are found, it fails.
+
+    * The structure of the `config/justfiles/` folder does not matter, folders/files can be placed in there however desired, the module will find all `.just` files.
+
+    * Optionally, the `.just` files can be validated.
```

Comment:
Why optionally? Why not validate by default, and only allow an opt-out in case there's a bug in the validation?

Reply:
> The validation can be very unforgiving and is turned off by default.

Ahh, so the validation isn't actually validation, but linting? So "invalid" justfiles still work, but there's just some minor issue?

Reply:
That's right. It's one of the reasons why we ditched just syntax checker action.